### PR TITLE
Translate PySyft Tutorials to Portuguese

### PR DESCRIPTION
**Pull request by hericlesme**
Translated notebooks:

- [x] Part 12 - Train an Encrypted Neural Network on Encrypted Data.ipynb
- [x] Part 13b - Secure Classification with Syft Keras and TFE - Secure Model Serving.ipynb

Issue #2776

**Note: The build is failing because it cannot find the `short-conv-mnist.h5` file on Notebook _Part 13b_, but the file is a result of a `model.save` operation on the Notebook _Part 13a_, so it needs to be run first.**

### CODE REVIEW

**Comment by hericlesme** [comment]
The build is failing because it cannot find the `short-conv-mnist.h5` file on Notebook _Part 13b_, but the file is a result of a `model.save` operation on the Notebook _Part 13a_, so it needs to be run first.

File: examples/tutorials/translations/português/Parte 12 bis - Treinamento criptografado no MNIST.ipynb [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+    "# Não usamos todo o conjunto de dados para fins de eficiência, mas fique à vontade para aumentar esses números\n",
+    "n_train_items = 640\n",
+    "n_test_items = 640"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part X - Treinamento e Avaliação Seguros no MNIST\n",
```
Comment (joaolcaas) [inline examples/tutorials/translations/português/Parte 12 bis - Treinamento criptografado no MNIST.ipynb]: Part - Parte

File: examples/tutorials/translations/português/Parte 12 bis - Treinamento criptografado no MNIST.ipynb [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+    "args = Arguments()\n",
+    "\n",
+    "_ = torch.manual_seed(args.seed)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Aqui estão as importações do PySyft. Nós nos conectamos a dois workers remotos que são chamados `alice` e` bob` e solicitamos outro worker chamado `crypto_provider`, que fornece todas as primitivas criptográficas que precisamos."
```
Comment (joaolcaas) [inline examples/tutorials/translations/português/Parte 12 bis - Treinamento criptografado no MNIST.ipynb]: add a space beteween ' ` ' and 'e'

File: examples/tutorials/translations/português/Parte 12 bis - Treinamento criptografado no MNIST.ipynb (hunk 1 above)
Comment (hericlesme) [reply examples/tutorials/translations/português/Parte 12 bis - Treinamento criptografado no MNIST.ipynb]: Oh, I forgot about this "Part", haha

File: examples/tutorials/translations/português/Parte 12 bis - Treinamento criptografado no MNIST.ipynb (hunk 2 above)
Comment (hericlesme) [reply examples/tutorials/translations/português/Parte 12 bis - Treinamento criptografado no MNIST.ipynb]: Done!

**Pull request merged**